django/template/base.py

```diff
@@ -786,7 +788,7 @@ def render(self, context):
         bits = []
         for node in self:
             if isinstance(node, Node):
-                bit = self.render_node(node, context)
+                bit = node.render_node(context)
```

Comment:
I might need a better name here, but ``render_node`` is correct. It's a wrapper around ``node.render`` that annotates debug information if there is an exception. I moved ``render_node`` to ``Node`` from ``NodeList`` because having it accessible on ``Node`` cleans up a hack in the ``ForNode``.